asv_bench/benchmarks/benchmarks.py

```diff
@@ -359,3 +359,29 @@ def setup(self, shape, columns_number, ascending_list):
 
     def time_sort_values(self, shape, columns_number, ascending_list):
         execute(self.df.sort_values(self.columns, ascending=self.ascending))
+
+
+class TimeFillna:
+    param_names = ["shape", "limit"]
+    params = [UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE], [None, 0.8]]
+
+    def setup(self, shape, limit):
+        columns = [f"col{x}" for x in range(shape[1])]
+
+        if ASV_USE_IMPL == "modin":
+            self.df = pd.DataFrame(
+                np.nan, index=pd.RangeIndex(shape[0]), columns=columns
+            )
+        elif ASV_USE_IMPL == "pandas":
+            self.df = pandas.DataFrame(
+                np.nan, index=pandas.RangeIndex(shape[0]), columns=columns
+            )
+        else:
+            raise NotImplementedError
```

Comment:
can we move logic about implementation dispatching to some function in utils? It would obtain all parameters to create a DataFrame + `"implementation"` parameter